cpataltmetrik/webdriveriowithtypescript
# Feature/selenoid integration

### Ticket Number / Feature:
Implementation of cross browser testing.
Segregated conf files to support cross browsers.

### Description:
Renamed wdio conf file to base file.
Created local conf file for local browsers like chrome, firefox and edge
2 conf files are created for seleniod
for headless execution, created a conf for it too.

## Requirements


## Code Formatting


## Best Practices


## Maintainability


## Documentation


## Testing


## Architecture


## Performance


1. For more information in detail on the checklist, go through this document - https://o365altimetrik-my.sharepoint.com/:w:/r/personal/cparihar_altimetrik_com/Documents/Code%20Review%20Checklist.docx?d=wb7d05100abc045dfa1af61e558ce71e2&csf=1&web=1

2. For better naming conventions, please go through the read me document.


### Test Execution Proof:
Ran the test cases will the conf files. Able to run them successfully.

[0-0] BASE URL : https://www.amazon.in
[0-0] RUNNING in chrome - D:\User\bpampari\automation\webdriveriowithtypescript\test\specs\forgotPassword.spec.ts

[0-0] BASE URL : https://www.amazon.in
[0-0] RUNNING in firefox - D:\User\bpampari\automation\webdriveriowithtypescript\test\specs\forgotPassword.spec.ts

[0-0] BASE URL : https://www.amazon.in
[0-0] RUNNING in MicrosoftEdge - D:\User\bpampari\automation\webdriveriowithtypescript\test\specs\forgotPassword.spec.ts

### Lint Result Proof:
NA

### How to Execute Test Case:
```
1. Set BROWSER value to chrome or firefox as per requirement
2. Run the command npm run test-dev for local exeution.
3. Check the execution results.
```
